pulp_container/app/registry_api.py

```diff
@@ -938,6 +942,10 @@ def put(self, request, path, pk=None):
         repository.pending_blobs.add(blob)
         return BlobResponse(blob, path, 201, request)
 
+    def _verify_payload_size(self, distribution, chunk):
+        if distribution.max_payload_size and chunk.reader.length > distribution.max_payload_size:
+            raise PayloadTooLarge()
```

Comment:
Is this exception properly rendered to the container's API format?

ref: https://github.com/pulp/pulp_container/blob/25abfd6cb0bff1ac8756e4d15c65b83839594f66/pulp_container/app/registry_api.py#L266

Reply:
Some of the exceptions are properly documented at https://docker-docs.uclv.cu/registry/spec/api/#blob-upload.

Reply:
Ahn... my bad, I didn't know about the spec for errors.
I also found this doc https://github.com/opencontainers/distribution-spec/blob/v1.0.1/spec.md#error-codes that will be helpful.